packages/wrangler/src/match-tag.ts

```diff
@@ -48,9 +49,17 @@ export async function verifyWorkerMatchesCITag(
 			throw new FatalError(
 				`The name in your ${configFileName(configPath)} file (${workerName}) must match the name of your Worker. Please update the name field in your ${configFileName(configPath)} file.`
 			);
+		} else if (e instanceof APIError) {
+			throw new FatalError(
+				"An error occurred while trying to verify the Worker.\n" +
```

Comment:
`while trying to verify the Worker` what does verify mean in this context? Will users understand what this is about?

Reply:
Yeah good point, I thought I was copying an existing error but now I don't know where I saw it haha

Updating the wording to match the other one which is clearer